roles/manage-jira/tasks/create_project.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Create Sample Project
+  uri:
+    url: "{{ atlassian.jira.url }}/rest/api/2/project"
+    method: POST
+    user: "{{ atlassian.jira.username }}"
+    password: "{{ atlassian.jira.password }}"
+    return_content: yes
+    force_basic_auth: yes
+    body_format: json
+    header:
+      - Accept: 'application/json'
+      - Content-Type: 'application/json'
+    body: "{ 'key': '{{ atlassian.jira.key }}',
+             'name': '{{ atlassian.jira.project }}',
+             'projectTypeKey': '{{ atlassian.projectTypeKey | default('software') }}',
```

Comment:
Looks like `jira` is missing in this usage of the fact - i.e. should be: `atlassian.jira.projectTypeKey`

Reply:
same for the one below